src/components/datePickerIOSRe.re

```diff
@@ -38,19 +45,7 @@ let make
             "onDateChange": from_opt onDateChange,
             "maximumDate": from_opt maximumDate,
             "minimumDate": from_opt minimumDate,
-            "mode":
-              from_opt (
-                UtilsRN.option_map
-                  (
-                    fun prop =>
-                      switch prop {
-                      | `date => "date"
-                      | `time => "time"
-                      | `datetime => "datetime"
-                      }
-                  )
-                  mode
-              ),
+            "mode": from_opt (UtilsRN.option_map encodeMode mode),
```

Comment:
Asking on Discord if a function allocation happens indeed. If not, I'd rather leave it the way it was before